styleguide/source/assets/scss/01-atoms/_display-switch.scss

```diff
@@ -0,0 +1,22 @@
+.ama__display-switch {
+  display: inline-block;
+  border: solid 1px $purple;
+  border-radius: 5px;
```

Comment:
should this be a variable so we can reuse it again?

Reply:
I don't know of anywhere else we use border radius at the moment, so I'm inclined to say no, but open to adding it to tech debt / addressing it later if something else comes up?